tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 skipsdist = True
-envlist = py3,pep8,pylint,bashate,functional
+envlist = py3-{non-utc-tz},pep8,pylint,bashate,functional
```

Comment:
is the curly brackets needed here?

Reply:
so as it is now, if i run ```tox``` it does:

  py3-non-utc-tz: commands succeeded
  pep8: commands succeeded
  pylint: commands succeeded
  bashate: commands succeeded
  functional: commands succeeded
  congratulations :)

i.e. it does not run -epy3. Personally i think it would be better to run -epy3-non-utc-tz in the github workflow (along with -epy3) and leave the default envlist as it was i.e. ```envlist = py3,pep8,pylint,bashate,functional```

Reply:
I see, I'll fix it in a few.

Reply:
... done